charts/redpanda/templates/tests/test-schemaregistry-internal-tls-status.yaml

```diff
@@ -19,10 +19,11 @@ limitations under the License.
   {{- $cert := get .Values.tls.certs $service.tls.cert -}}
   {{- $root := deepCopy . }}
   {{- $sasl := .Values.auth.sasl }}
+  {{- $randNumber := randNumeric 3 }}
 apiVersion: v1
 kind: Pod
 metadata:
-  name: {{ include "redpanda.fullname" . }}-test-schemaregistry-internal-tls-status
+  name: {{ include "redpanda.fullname" . }}-test-sr-internal-tls-status-{{ $randNumber }}
```

Comment:
was this causing collissions? the hooks should have removed this on installs.

Reply:
I had problem with showing in the following scenario

1. helm install
2. helm test
3. helm upgrade
4. helm test

Schema registry test in point 2 execution is important in the bug report. 

If pod could be changed to Job, maybe I could have both execution point 2 and 4.